examples/cookie/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const i18n = require('hmpo-i18n');
+
+// Visit localhost:3000/language/cy to change to Welsh using cookies
+// Visit localhost:3000/language/en to change to English using cookies
+
+// Setup middleware
+app.use(cookieParser());
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, 'views'));
+
+// Create translator instance
+const Translator = i18n.Translator;
+const translator = new Translator({
+    resources: {
+        en: require('./locales/en/default.json'),
+        cy: require('./locales/cy/default.json')
+    }
+});
+
+// Create our own middleware
+app.use((req, res, next) => {
+    // Get language from query param or cookie or default to English
+    const userLang = (req.query && req.query.lang && ['en', 'cy'].includes(req.query.lang))
```

Comment:
This isnt using the cookie option that can be passed in - the example should be something to do with using the cookie option if you are going to include it